pkg/perfcounter/counter_logexporter.go

```diff
@@ -52,5 +54,11 @@ func (c *CounterLogExporter) Export() []zap.Field {
 
 	fields = append(fields, zap.Any("disk hit rate", float64(diskHits)/float64(diskReads)))
```

Comment:
Should we remove the old appends? ie

```go
        fields = append(fields, zap.Any("reads", reads))
	fields = append(fields, zap.Any("hits", hits))
	fields = append(fields, zap.Any("hit rate", float64(hits)/float64(reads)))
	fields = append(fields, zap.Any("mem reads", memReads))
	fields = append(fields, zap.Any("mem hits", memHits))
	fields = append(fields, zap.Any("mem hit rate", float64(memHits)/float64(memReads)))
	fields = append(fields, zap.Any("disk reads", diskReads))
	fields = append(fields, zap.Any("disk hits", diskHits))

	fields = append(fields, zap.Any("disk hit rate", float64(diskHits)/float64(diskReads)))
```

Reply:
Right now we don't have the option to load the current window counter value from `stats.Counter`. We only have the option for `Swap` to fetch the current [window counter value](https://github.com/matrixorigin/matrixone/blob/557c8ca22c63ec0fa6f51a6f8331348edaa8079a/pkg/util/metric/stats/counter.go#L32). I think, it is now a necessity to introduce LoadW(), which loads the current window counter value.

```go
func (c *Counter) LoadW() int64 {
	return c.window.Load()
}
```

I think, once that is implemented inside `stats.Counter`, we can keep the rate fields outside of IterFields function. ie

```go
fields = append(fields, zap.Any("FileService Cache Hit Rate", float64(c.counter.FileService.Cache.Hit.LoadW())/float64(c.counter.FileService.Cache.Read.LoadW())))
fields = append(fields, zap.Any("FileService Cache Memory Hit Rate", float64(c.counter.FileService.Cache.Memory.Hit.LoadW())/float64(c.counter.FileService.Cache.Memory.Read.LoadW())))
fields = append(fields, zap.Any("FileService Cache Disk Hit Rate", float64(c.counter.FileService.Cache.Disk.Hit.LoadW())/float64(c.counter.FileService.Cache.Disk.Read.LoadW()))
```

and remove the rest of the manual appends.

ie
```go
        fields = append(fields, zap.Any("reads", reads))
	fields = append(fields, zap.Any("hits", hits))
	fields = append(fields, zap.Any("mem reads", memReads))
	fields = append(fields, zap.Any("mem hits", memHits))
	fields = append(fields, zap.Any("disk reads", diskReads))
	fields = append(fields, zap.Any("disk hits", diskHits))
```

as it will be handled by `IterFields()`

Reply:
Done